spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkView.java

```diff
@@ -36,7 +36,7 @@
 public class SparkView implements org.apache.spark.sql.connector.catalog.View {
 
   private static final String QUERY_COLUMN_NAMES = "queryColumnNames";
```

Comment:
Is this our property name? We never use camelCase for Iceberg properties. I think this should be `query-column-names` and should possibly have a prefix, like `spark.query-column-names`.

Reply:
good point, I've renamed this to `spark.query-column-names`